python/test/extras/test_ext_domains.py

```diff
@@ -25,6 +23,9 @@ def test_array2_domain():
     # origin array dtype must be numeric
     with pytest.raises(ValueError):
         dp.numpy.array2_domain(norm=1, p=2, origin=np.array([True, False]))
+    
+    with pytest.raises(ValueError):
```

Comment:
Should we lean towards making an assertion about the content of the message? It might introduce some fragility to the tests, but if we're going to spend some time making the tests more useful, we need to test those messages to make sure we don't regress.